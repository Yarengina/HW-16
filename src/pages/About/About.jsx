// eslint-disable-next-line no-unused-vars
import * as React from 'react'
import './About.css'
import Navigation from '../../components/Navigation/Navigation'

function About() {
    return (
        <>
            <Navigation />
            <div className="About">
                <h2>О магазине</h2>
                <p>Магазин книг</p>
            </div>
        </>
    )
}

export default About
