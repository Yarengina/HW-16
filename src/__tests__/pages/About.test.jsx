// eslint-disable-next-line no-unused-vars
import * as React from 'react'
import { render } from '../utils/renderWithRouter'
import About from '../../pages/About/About'

describe('<About />', () => {
    it('should render successfully and match snapshot', () => {
        const { container } = render(<About />)
        expect(container).toMatchSnapshot()
    })
})
