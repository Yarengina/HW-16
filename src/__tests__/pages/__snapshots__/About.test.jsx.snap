// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<About /> should render successfully and match snapshot 1`] = `
<div>
  <nav
    class="LinkField"
  >
    <a
      aria-current="page"
      class="Link active"
      data-testid="login-link"
      href="/"
    >
      <span>
        Login
      </span>
    </a>
    <a
      class="Link"
      data-testid="about-link"
      href="/about"
    >
      <span>
        About
      </span>
    </a>
    <a
      class="Link"
      data-testid="order-link"
      href="/order"
    >
      <span>
        Order
      </span>
    </a>
  </nav>
  <div
    class="About"
  >
    <h2>
      О магазине
    </h2>
    <p>
      Магазин книг
    </p>
  </div>
</div>
`;
